addons/crm/tests/test_crm_lead.py

```diff
@@ -423,6 +423,24 @@ def test_crm_lead_stages(self):
         self.assertEqual(lead.probability, 100.0)
         self.assertEqual(lead.stage_id, self.stage_gen_won)  # generic won stage has lower sequence than team won stage
 
+    @users('user_sales_manager')
+    def test_crm_last_stage_update_time(self):
```

Comment:
Just make sure to move this to be in alphabetical order :)
(order in which the tests execute, and makes it easier to find)